serial_communication/web/app.py

```diff
@@ -160,6 +160,12 @@ def send_auth_route():
     return send_auth()
 
 
+@app.route('/send_sms', methods=['GET'])
+def send_custom_sms():
+    """Send custom message."""
+    print ("\n\ngot you homie?\n")
```

Comment:
hope you don't push this on purpose, please remove this and let any of us know so we can continue with PR approval.